theworkoutinfo/Coordinators/HomeCoordinator.swift

```diff
@@ -29,12 +29,12 @@ class HomeCoordinator: BaseCoordinator<HomeCoordinatorResult> {
         window.rootViewController = navigationController
         window.makeKeyAndVisible()
         
-        let logout = viewModel.didLogout.map { _ -> HomeCoordinatorResult in
-            UserDefaults.standard.set(false, forKey: "isUserLoggedIn")
+        let didLogout = viewModel.didLogout.map { _ -> HomeCoordinatorResult in
+            AuthenticationRepository.logout()
```

Comment:
Coordinators shouldn't call this logic. Let the viewModel call it and then answer to the coordinator